Gems/ROS2/Code/Source/Imu/ROS2ImuSensorComponent.cpp

```diff
@@ -102,7 +102,15 @@ namespace ROS2
 
     void ROS2ImuSensorComponent::SetupRefreshLoop()
     {
-        InstallPhysicalCallback(m_entity->GetId());
+        auto entityId = m_entity->GetId();
```

Comment:
I do not really like that. A number of sensors require a handler to rigid body / simulated body. The problem with that is:
- the code for taking rigid body handle is duplicated (even in this PR).
- this API is used outside of the project and it is breaking change.